app/templates/environmental-solutions.hbs

```diff
@@ -0,0 +1,21 @@
+<div class="header">
+    <h1>Environmental Solutions</h1>
+</div>
+<div class="content">
+    <form class="pure-form search" {{action "search" on="submit"}}>
+        <fieldset>
+            <div class="pure-g">
+                <div class="pure-u-1">
+                    <label>Keyword</label>
+                  {{input class="pure-input-1" value=qField}}
```

Comment:
I thought this was a special name related to Ember, i think i know now what is going on.